### ISSUE TITLE
Deal with conflicting input CSVs

### ISSUE THREAD

**Issue by alexwalkercebm** [opensafely/research-template#4]
After making a change to the study definition in a PR, the `input.csv` then conflicts with the one in `master`. This makes it difficult to merge. I guess we always want to keep the more recently generated version.
![image](https://user-images.githubusercontent.com/24895935/84039133-b715be00-a998-11ea-9500-bd485ee8a944.png)

**Comment by sebbacon** [comment]
This is tricky. @inglesp's idea seems the best: have branches generate a differently-named file (`temp_branch_input.csv`?) which is deleted on merge; and have the "real" one generated in master on merge.

**Comment by sebbacon** [comment]
Hmm. Having started this, it feels a bit messy.

I think perhaps making the CSVs release assets might be the best thing.

We can make `latest` always be current master so it can be linked to from the README.

We can also a per-branch release which is deleted when the branch is merged.

It's ugly but we should be encouraging most users to use the command line anyway.

It means we don't end up polluting the git history with meaningless diffs too.

**Comment by sebbacon** [comment]
* [Example release builder](https://github.com/ebmdatalab/opensafely-research-template/blob/20c468137bc0ecd4edfe28a71cbe5ff2b02836f6/.github/workflows/windows_build_and_release.yaml)
* [Example release deleter](https://github.com/ebmdatalab/opensafely-research-template/blob/20c468137bc0ecd4edfe28a71cbe5ff2b02836f6/.github/workflows/delete_windows_prerelease.yaml)

**Comment by alexwalkercebm** [comment]
Yes that does sound better, and would mean there's less restriction on the size of dummy data population.
